warehouse/core/src/main/java/datawave/iterators/filter/ConfigurableAgeOffFilter.java

```diff
@@ -346,7 +347,7 @@ private void initFilterRules() throws IllegalArgumentException, IOException {
                                     .expireAfterAccess(EXPIRATION_INTERVAL_MS, TimeUnit.MILLISECONDS).build(new ReloadableCacheBuilder());
                     // this will schedule a check to see if the update or expiration intervals have changed
                     // if so the ruleCache will be rebuilt with these new intervals
-                    SimpleTimer.getInstance(AccumuloConfiguration.getDefaultConfiguration()).schedule(
+                    SimpleTimer.getInstance(1).schedule(
```

Comment:
Since the default is "1" for the thread pool size, I made this call the other SipmleTimer method which just takes an integer for the thread pool size.  SimpleTimer hasn't changed but AccumuloConfiguration has changed.